calcom/cal.com
# fix: fixes issue with webhook not being triggered for managed events

## What does this PR do?

This PR fixes the issue that the Booking Created webhook was not being triggered for managed events. The reason was that the eventTypes created for every assigned team member did not contain the teamId which was needed to find the right webhook while triggering.

We can't add teamId to this eventType either because [slug,teamId] is a unique identifier for the event types and it's important that it stays that way so that we can't have a team having multiple events with the same slug. So what I've done is while booking I'll check whether it's a managed event and then will extract the eventType from the eventType of this slug which contains the teamId this way we will be able to access the teamId.

Also, the event type that was being created when a team member was assigned a managed event did not contain the schedulingType so have added that too.

Fixes #9962 



https://github.com/calcom/cal.com/assets/47187878/19ffda33-0362-4fbf-82a4-7e188495c94e


## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?



- Create a managed event 
- Configure a webhook for this event with BOOKING.CREATED trigger enabled
- Add members to this event
- Make a booking
- Check that the webhook was triggered

## Mandatory Tasks
